django/core/management/commands/loaddata.py

```diff
@@ -251,6 +251,27 @@ def _get_targets(self, fixture_name, ser_fmt, cmp_fmt):
         )
         return {'.'.join((fixture_name, suffix)) for suffix in suffixes}
 
+    def _find_fixture_files_in_dir(self, fixture_dir, fixture_name, targets):
+        fixture_files_in_dir = []
+        path = os.path.join(fixture_dir, fixture_name)
+        for candidate in glob.iglob(glob.escape(path) + '*'):
+            if os.path.basename(candidate) in targets:
+                # Save the fixture_dir and fixture_name for future error messages.
+                fixture_files_in_dir.append((candidate, fixture_dir, fixture_name))
+        return fixture_files_in_dir
+
+    def _check_fixture_files_in_dir(self, fixture_dir, fixture_name, fixture_files_in_dir):
```

Comment:
I don't see much value in adding `_check_fixture_files_in_dir()`. I would move this to the `_find_fixture_files_in_dir()` or leave it in the main loop.

Reply:
Done in e83e4694ef21e2b4bb7573cb98c2bf56f97c05d0